### ISSUE TITLE
Build error importing TrustKit.h into .mm file

### ISSUE THREAD

**Issue by cohenadair** [datatheorem/TrustKit#160]
Importing TrustKit into an Objective-C++ (.mm) file via `#import <TrustKit/TrustKit.h>` results in the following build error:
```
Use of '@import' when C++ modules are disabled, consider using fmodules and fcxx-modules
```
From what I've read, C++ modules are disabled for Objective-C++ files, and the `@import` directive isn't allowed.

This seems to be a common problem in Pod libraries, other's have fixed it by doing something like:
```Objective-C
#if __has_feature(modules)
    @import Foundation;
#else
    #import <Foundation/Foundation.h>
#endif
```

**For others having this issue**

A workaround is to create a `TrustKitWrapper.h/m` class and create wrapper methods for TrustKit APIs you need, then import `TrustKitWrapper.h` into your `.mm` files. Something like:
```Objective-C
@interface TrustKitWrapper : NSObject

+ (void)setupTrustKitInstance;
+ (bool)handleChallenge:(NSURLAuthenticationChallenge *)challenge;

@end
```

**Comment by boehlefeld** [comment]
Thanks @cohenadair , I do have the same issue here tryin to import TrustKit in Cedar BDD Testing. How about making a PR?

**Comment by cohenadair** [comment]
@boehlefeld I'm not likely to have the time anytime soon, but I'll add it to my list of things to do. :)

**Comment by nabla-c0d3** [comment]
Released with v1.6.0.
